cmd/mattermost/commands/config.go

```diff
@@ -251,10 +251,23 @@ func configSetCmdF(command *cobra.Command, args []string) error {
 		return errors.New("Invalid locale configuration")
 	}
 
-	if _, err := configStore.Set(newConfig); err != nil {
-		return errors.Wrap(err, "failed to set config")
+	if _, errSet := configStore.Set(newConfig); errSet != nil {
+		return errors.Wrap(errSet, "failed to set config")
 	}
 
+	/*
```

Comment:
What is the problem preventing this from running correctly?

Reply:
I created a ticket here https://mattermost.atlassian.net/browse/MM-23630 so I'll remember to come back and address this. 

When I added the app initialization code here it caused some unit tests to fail, but only on CI.  Locally it was fine.  I spend quite a bit of time trying to figure out what was wrong, with "experiment" and "attempt X" commits, but each iteration takes a long time for CI to even spin up.  I was going to try and replicate CI locally when I attempt to fix this.

Reply:
Makes sense, thanks for the info!